index.html

```diff
@@ -2470,15 +2466,30 @@ <h2>Production and Consumption</h2>
 
       <ol>
         <li>
-A <a>conforming consumer</a> MUST add all entries that do not have explicit
-processing rules for the <a>representation</a> being consumed to the <a
-href="#data-model">data model</a> using only the <a>representation</a>'s data
-type processing rules.
+A <a>conforming consumer</a> MUST take a <a>representation</a> and
+MIME type <a data-cite="INFRA#string">string</a> as input into
```

Comment:
```suggestion
Media Type <a data-cite="INFRA#string">string</a> as input into
```
or "media type" ... Capitalization may be important for clarity.

Reply:
Rejecting per https://github.com/w3c/did-core/pull/679/files#r579329532

Reply:
Fixed in d915bea303b36c00ec59b27b8ac10845da828fea